prettier.py

```diff
@@ -35,21 +35,30 @@ def report_error(returncode, err=None):
     if err:
         print(err)
 
+# Calls prettier on the given region and replaces code
```

Comment:
Two empty lines above function defs please. It's not written anywhere yet but the project is following flake8 linting rules.